tests/model_hub_tests/transformation_tests/sdpa2pa_ref_diff.py

```diff
@@ -0,0 +1,595 @@
+nodes_to_compare = ("ScaledDotProductAttention", "PagedAttentionExtension", "Parameter", "ReadValue", "Assign")
+
+ref_diff_map = {
+	"hf-internal-testing/tiny-random-LlamaForCausalLM" : {
+		"PagedAttentionExtension" : 2,
+		"ScaledDotProductAttention" : -2,
+		"Parameter" : 7,
+		"ReadValue" : -4,
+		"Assign" : -4,
```

Comment:
can you please give comments where are these values coming from?

Reply:
These values were generated using generate_model_diffs.py. If someone's making a change that would introduce some change in the number of nodes, the values can be regenerated.